Dockerfile

```diff
@@ -1,22 +1,146 @@
 FROM airdock/oraclejdk:1.8 as metabrainz-spark-base
 
-RUN printf "deb http://archive.debian.org/debian/ jessie main\ndeb-src http://archive.debian.org/debian/ jessie main\ndeb http://security.debian.org jessie/updates main\ndeb-src http://security.debian.org jessie/updates main" > /etc/apt/sources.list
+# Compile and install specific version of Python
+# The jdk image comes with jessie which has python 3.4 which
+# is not supported anymore
```

Comment:
might want to additionally say here that we're installing 3.6 because of openssl, especially since the following line says 3.7